runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1665,32 +1665,121 @@ static void buildVirtualCall(TR::CodeGenerator *cg, TR::Node *callNode, TR::Regi
 /**
  * @brief Generates instruction sequence for interface call
  *
- * @param[in] cg:          code generator
- * @param[in] callNode:    node for the interface call
- * @param[in] vftReg:      vft register
- * @param[in] tmpReg:      temporary register
- * @param[in] tmp2Reg:     temporary register
- * @param[in] ifcSnippet:  interface call snippet
- * @param[in] regMapForGC: register map for GC
+ * @param[in] cg:                 code generator
+ * @param[in] callNode:           node for the interface call
+ * @param[in] vftReg:             vft register
+ * @param[in] x9Reg:              temporary register
+ * @param[in] tmpReg:             temporary register
+ * @param[in] tmp2Reg:            temporary register
+ * @param[in] useLastITableCache  if true, use last iTable cache
```

Comment:
A colon (`:`) is missing after `useLastITableCache` in the comment.

Reply:
Added a colon.